add forecast_ILI to incidencemapR which uses ARIMA to forecast ILI - …

grhuynh opened the pull request:
…returns the forecasted ILI with 95% CI and saves a pdf graph of the ILI forecast

grhuynh committed:
add forecast_ILI to incidencemapR which uses ARIMA to forecast ILI - returns the forecasted ILI and saves a pdf graph of the ILI forecast

famulare commented:
Thanks Grace!  I'll go through this next week and let you know if there are any issues.

famulare committed:
many changes to make forecast_ILI more compatible with rest of codebase
- replaced non-tidyverse data manipulation packages with tidy equivalents
- added explicit namespace references where not clear
- added newly required packages to Dockerfile.RBuildEnv
- converted all week variable to iso 8601 standard
- used same pattern for forecast week extension as in expandDB.R
- made all references to current date for forecasting explicit instead of implicit differences

There are still things that should be further cleaned up for stability and reuse.  Such as:
- make references to WA_ILI configurable.
- more descriptive variable names
- integration into dbViewR workflow for appending ILI to data.
- ...

famulare committed:
removed unecessary reshape2 import

famulare commented:
This request was superceded by 1bdf8483b9f701b0a76c3da381e24cc8e555107b
